openapi.json

```diff
@@ -171,6 +175,35 @@
               "type": "string"
             },
             "description": "The recommendation UUID"
+          },
+          {
```

Comment:
@saltgen  - could you check here - https://swagger.io/docs/specification/describing-parameters/#default and see if it helps to present default value for these params?

Reply:
Yep, possible via the enum type, changes pushed